src/middlewared/middlewared/plugins/nfs.py

```diff
@@ -609,7 +614,7 @@ async def validate_hosts_and_networks(self, other_shares, data, schema_name, ver
             if host_ip is None:
                 verrors.add(
                     f"{schema_name}.hosts",
-                    f"Unable to resolve host {host}"
+                    f"Unable to resolve host '{host}'"
```

Comment:
Just an GYI but you can add !r before the right curly brace and the f-string formatting will single-quite it for you.